config/webpack.config.js

```diff
@@ -175,21 +179,11 @@ const configGenerator = (options, apps) => {
     },
     plugins: [
       new webpack.DefinePlugin({
-        __BUILDTYPE__: JSON.stringify(options.buildtype),
-        'process.env': {
-          API_PORT: process.env.API_PORT || 3000,
-          WEB_PORT: process.env.WEB_PORT || 3333,
-          API_URL: process.env.API_URL
-            ? JSON.stringify(process.env.API_URL)
-            : null,
-          BASE_URL: process.env.BASE_URL
-            ? JSON.stringify(process.env.BASE_URL)
-            : null,
-        },
```

Comment:
These can all for different reasons -

`process.env.API_PORT` and `process.env.WEB_PORT` - were only available for use potentially in our [E2E config](https://github.com/department-of-veterans-affairs/vets-website/blob/9fad755e49b0a3d8d271ea0dc710f17ef2426ef5/src/platform/utilities/environment/index.js#L22). However, they weren't actually useful at all, because Nightwatch-Docker tests (which execute on Jenkins) were executing under port `3001`, which caused this [condition](https://github.com/department-of-veterans-affairs/vets-website/blob/9fad755e49b0a3d8d271ea0dc710f17ef2426ef5/src/platform/utilities/environment/index.js#L32) to take effect instead. So in theory we could have been using these environment variables locally, but it seems like needless complexity.

`process.env.BASE_URL` and `process.env.API_URL` are never processed anywhere in the Webpack or Metalsmith code.